ddtrace/contrib/django/conf.py

```diff
@@ -179,7 +179,7 @@ def configure_from_settings(pin, config, settings):
             setattr(config, config_key, settings[setting_key])
 
     # re-configure tracer
-    if "TRACE" in settings:
+    if "TRACER" in settings:
         pin.tracer = settings["TRACER"]
```

Comment:
It looks like when the above condition is true this triggers another bug `pin.tracer = settings["TRACER"]`

but... maybe it doesn't matter since `configure_from_settings` appears to be deprecated anyway?

Reply:
`AttributeError: can't mutate a pin, use override() or clone() instead`